src/treemodel.cpp

```diff
@@ -91,6 +91,26 @@ QVariant TreeModel::data(const QModelIndex &index, int role) const
             return item->Data(col);
             break;
         }
+        case Qt::ToolTipRole:
+        {
+            if (col == COL::Value)
+            {
+                const int ToolTipLimit = 500;
```

Comment:
We have 500 for events as well and I assumed we want them aligned, right?
(One with line breaks and other without)